server/coordinator/coordinator_request.go

```diff
@@ -8,13 +8,95 @@ import (
 	"github.com/suborbital/appspec/request"
 	"github.com/suborbital/appspec/tenant"
 	"github.com/suborbital/appspec/tenant/executable"
-	"github.com/suborbital/deltav/scheduler"
-	"github.com/suborbital/deltav/server/coordinator/sequence"
+	"github.com/suborbital/e2core/scheduler"
+	"github.com/suborbital/e2core/server/coordinator/sequence"
 	"github.com/suborbital/vektor/vk"
 )
 
-func (c *Coordinator) vkHandlerForModule(mod tenant.Module) vk.HandlerFunc {
+func (c *Coordinator) vkHandlerForModuleByName() vk.HandlerFunc {
 	return func(r *http.Request, ctx *vk.Ctx) (interface{}, error) {
+		ident := ctx.Params.ByName("ident")
+		namespace := ctx.Params.ByName("namespace")
+		name := ctx.Params.ByName("name")
+
+		tnt := c.syncer.TenantOverview(ident)
+		if tnt == nil {
+			return nil, vk.E(http.StatusNotFound, "tenant not found")
+		}
+
+		var mod *tenant.Module
+
+		for i, m := range tnt.Config.Modules {
```

Comment:
would it make sense to move module discovery into the syncer instead? 

i.e. `c.sycner.GetModule(ctx.Context, ident, namespace, name)`

Everyone may not agree but that feels like a more clean abstraction and is a bit more readable since all our SystemSource code is handled there now.